connectors/citrus-docker/src/test/java/org/citrusframework/docker/actions/dsl/DockerTestActionBuilderTest.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2006-2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.citrusframework.docker.actions.dsl;
+
+import java.util.UUID;
+
+import com.github.dockerjava.api.command.CreateContainerCmd;
+import com.github.dockerjava.api.command.CreateContainerResponse;
+import com.github.dockerjava.api.command.InfoCmd;
+import com.github.dockerjava.api.command.InspectContainerCmd;
+import com.github.dockerjava.api.command.InspectContainerResponse;
+import com.github.dockerjava.api.command.PingCmd;
+import com.github.dockerjava.api.command.VersionCmd;
+import com.github.dockerjava.api.model.Info;
+import com.github.dockerjava.api.model.Version;
+import org.citrusframework.DefaultTestCaseRunner;
+import org.citrusframework.TestCase;
+import org.citrusframework.docker.UnitTestSupport;
+import org.citrusframework.docker.actions.DockerExecuteAction;
+import org.citrusframework.docker.client.DockerClient;
+import org.mockito.Mockito;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import static org.citrusframework.docker.actions.DockerExecuteAction.Builder.docker;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.reset;
+import static org.mockito.Mockito.when;
+
+/**
+ * @author Christoph Deppisch
+ * @since 2.4
+ */
+public class DockerTestActionBuilderTest extends UnitTestSupport {
```

Comment:
I think test classes, as well as methods, should not be declared public.

Reply:
Yeah, thanks. Probably something to add to the unit test refactoring task that targets to use JUnit Jupiter instead of TestNG